src/client/resources/conversions.ts

```diff
@@ -472,12 +472,15 @@ export function updateMigPlanFromValues(
 
       return updatedPV;
     });
-    updatedSpec.persistentVolumes = planValues.persistentVolumes;
+    updatedSpec.persistentVolumes = _.unionBy(
+      planValues.persistentVolumes,
+      updatedSpec.persistentVolumes
+    );
```

Comment:
I think this needs to be:
```suggestion
    updatedSpec.persistentVolumes = _.unionBy(
      [planValues.persistentVolumes, updatedSpec.persistentVolumes],
      (pv) => pv.metadata.name
    );
```